content/en/cosign/signing_with_blobs.md

```diff
@@ -95,3 +48,51 @@ tlog entry created with index: 965333
 Signature wrote in the file sig
 Certificate wrote in the file cert.pem
 ```
+
+
+## Signing with a key
+
+You may specify your own keys for signing.  You will need the password for the private key to sign:
```

Comment:
Does it make sense to say something here recommending using keyless signing, or at least linking to why that's recommended?